manifester/helpers.py

```diff
@@ -120,6 +124,34 @@ def fetch_paginated_data(manifester, endpoint):
         return _endpoint_data
 
 
+def load_inventory_file(file):
+    """Load local inventory file.
+
+    :return: list of dictionaries
+    """
+    if file.suffix in (".yaml", ".yml"):
+        loader = yaml
+        loader_args = {"Loader": yaml.FullLoader}
+        with file.open() as f:
+            return loader.load(f, **loader_args) or []
```

Comment:
if you're not expecting to load from json, you can just simplify this to be yaml-specific